src/Oscar.jl

```diff
@@ -80,9 +80,21 @@ function __init__()
   GAP.Globals.BindGlobal(GapObj("Oscar"), Oscar)
   GAP.Globals.SetPackagePath(GAP.Obj("OscarInterface"), GAP.Obj(joinpath(@__DIR__, "..", "gap", "OscarInterface")))
   GAP.Globals.LoadPackage(GAP.Obj("OscarInterface"))
+  # Work around a problem in Julia 1.10 (see https://github.com/oscar-system/Oscar.jl/pull/3222).
+  isdefined(GAP.Packages.Downloads, :default_downloader!) && GAP.Packages.Downloads.default_downloader!(GAP.Packages.Downloads.Downloader(grace=0.01))
```

Comment:
I have just release GAP.jl 0.10.2 so you could remove this and update `Project.toml` instead.

Reply:
Great. Thanks.